cartographer/mapping_3d/submaps.cc

```diff
@@ -198,8 +198,8 @@ proto::SubmapsOptions CreateSubmapsOptions(
 }
 
 Submap::Submap(const float high_resolution, const float low_resolution,
-               const transform::Rigid3d& local_pose)
-    : mapping::Submap(local_pose),
+               const transform::Rigid3d& local_submap_pose)
```

Comment:
Is this different than the local map frame?

Reply:
This is not a frame. The local submap pose according to terminology:
  Transforms data from a submap frame to the local map frame.